reference/3.0/Microsoft.PowerShell.Core/About/about_Remote_Disconnected_Sessions.md

```diff
@@ -112,59 +112,67 @@ The first command creates a session to the Server01 computer. The session
+New-PSSession -ComputerName Server01
+```
 
+```Output
 Id Name      ComputerName  State    ConfigurationName     Availability
 -- ----      ------------  -----    -----------------     ------------
  2 Session2  Server01      Opened   Microsoft.PowerShell     Available
 ```
 
-To get the session, use the **ComputerName** parameter of `Get-PSSession`
-with a value of Server01.
+To get the session, use the **ComputerName** parameter of `Get-PSSession` with
+a value of Server01.
 
 ```powershell
-PS> Get-PSSession -ComputerName Server01
+Get-PSSession -ComputerName Server01
+```
 
+```Output
 Id Name      ComputerName  State    ConfigurationName     Availability
 -- ----      ------------  -----    -----------------     ------------
  2 Session2  Server01      Opened   Microsoft.PowerShell     Available
 ```
 
-If the value of the **ComputerName** parameter of `Get-PSSession` is
-localhost, `Get-PSSession` gets PSSessions that terminate at and are
-maintained on the local computer. It does not get PSSessions on the Server01
-computer, even if they were started on the local computer.
+If the value of the **ComputerName** parameter of `Get-PSSession` is localhost,
+`Get-PSSession` gets PSSessions that terminate at and are maintained on the
+local computer. It doesn't get PSSessions on the Server01 computer, even if
+they were started on the local computer.
 
 ```powershell
-PS> Get-PSSession -ComputerName localhost
-PS>
+Get-PSSession -ComputerName localhost
 ```
 
 To get sessions that were created in the current session, use the
-`Get-PSSession` cmdlet without parameters. This command gets the PSSession
-that was created in the current session and connects to the Server01 computer.
+`Get-PSSession` cmdlet without parameters. In this example, `Get-PSSession`
+gets the PSSession that was created in the current session and connects to the
+Server01 computer.
 
 ```powershell
-PS> Get-PSSession
+Get-PSSession
+```
 
+```Output
 Id Name      ComputerName  State    ConfigurationName     Availability
 -- ----      ------------  -----    -----------------     ------------
  2 Session2  Server01      Opened   Microsoft.PowerShell     Available
 ```
 
-## How to Disconnect Sessions
+## How to disconnect sessions
 
-To disconnect a PSSession use the `Disconnect-PSSession` cmdlet. To identify
-the PSSession, use the **Session** parameter, or pipe a PSSession from the
+To disconnect a PSSession, use the `Disconnect-PSSession` cmdlet. To identify
+the PSSession, use the **Session** parameter, or pipeline a PSSession from the
 `New-PSSession` or `Get-PSSession` cmdlets to `Disconnect-PSSession`.
 
 The following command disconnects the PSSession to the Server01 computer.
-Notice that the value of the State property is Disconnected and the
-Availability is None.
+Notice that the value of the **State** property is **Disconnected** and the
+**Availability** is **None**.
 
 ```powershell
 PS> Get-PSSession -ComputerName Server01 | Disconnect-PSSession
```

Comment:
Probably can remove the PS prompt here as well

Reply:
Resolved.